Tasks/AzureFileCopyV1/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
-        "Patch": 151
+        "Minor": 153,
```

Comment:
We need the Patch version change in master as well, else this release branch version will be higher than the master versions.

Reply:
Resolved offline